blog/forms.py

```diff
@@ -0,0 +1,13 @@
+"""Forms for the blog app."""
+from django import forms
+from .models import Post
+
+
+class PostForm(forms.ModelForm):
+    """Create and edit form."""
```

Comment:
It's not an action, but entity.

Reply:
@webknjaz `Creating and editing forms.` ?  Is good doc for this `class`?